src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/tests/test_webapp_commands.py

```diff
@@ -832,7 +832,7 @@ def test_webapp_update(self, resource_group):
                      JMESPathCheckV2('clientAffinityEnabled', False)])
 
 
-class WebappZipDeployScenarioTest(ScenarioTest):
+class WebappZipDeployScenarioTest(LiveScenarioTest):
```

Comment:
Can I know what makes this test unrecordable?

Reply:
It involves a binary file which is not able to be serialized correctly by the recording. I did the same thing on the tests written for the log file downloading

Reply:
I see.

Reply:
Ok. Let's keep it as live.